backend/src/main/resources/liquibase/db-changelog-2.0.xml

```diff
@@ -993,4 +993,27 @@
         <tagDatabase tag="db_version_2.35"/>
     </changeSet>
 
+    <changeSet id="create-event-table" author="anandJ">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="event"/>
+            </not>
+        </preConditions>
+        <createTable tableName="event">
+            <column name="event_uuid" type="varchar(64)">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="pk_query_event_uuid"/>
+            </column>
+            <column name="event_type" type="varchar(64)"/>
+            <column name="workspace_id" type="int8"/>
```

Comment:
Doesn't an event _have_ to be correlated to a workspace id? I might say workspace id is not nullable.

Reply:
I don't feel strongly on this